packages/desktop-client/src/components/manager/subscribe/Login.tsx

```diff
@@ -86,27 +119,52 @@ export function Login() {
         </Text>
       )}
 
-      <form
-        style={{ display: 'flex', flexDirection: 'row', marginTop: 30 }}
-        onSubmit={onSubmit}
-      >
-        <BigInput
-          autoFocus={true}
-          placeholder="Password"
-          type="password"
-          onChange={(e: ChangeEvent<HTMLInputElement>) =>
-            setPassword(e.target.value)
-          }
-          style={{ flex: 1, marginRight: 10 }}
-        />
-        <ButtonWithLoading
-          type="primary"
-          loading={loading}
-          style={{ fontSize: 15 }}
+      {method === 'password' && (
+        <form
+          style={{ display: 'flex', flexDirection: 'row', marginTop: 30 }}
+          onSubmit={onSubmit}
         >
-          Sign in
-        </ButtonWithLoading>
-      </form>
+          <BigInput
+            autoFocus={true}
+            placeholder="Password"
+            type="password"
+            onChange={e => setPassword(e.target.value)}
```

Comment:
```suggestion
            onChange={(e: ChangeEvent<HTMLInputElement>) =>
              setPassword(e.target.value)
            }
```

To match the original